docs/guides/variables/aws/s3.md

```diff
@@ -0,0 +1,126 @@
+<!--
+title: Serverless Framework - Variables - S3 Objects
+description: How to reference AWS S3 Objects
+short_title: Serverless Variables - S3 Objects
+keywords: ['Serverless Framework', 'S3', 'AWS', 'Variables']
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/guides/variables/s3)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Configuration options
+
+| Option       | Required |  Type  |              Default               | Description        |
+| ------------ | :------: | :----: | :--------------------------------: | :----------------- |
+| `region`     |    No    | String | Inherited from parent AWS resolver | AWS region         |
+| `bucketName` |    No    | String |                                    | AWS S3 bucket name |
+| `objectKey`  |    No    | String |                                    | AWS S3 object key  |
+
+## Examples
+
+### Default
+
```

Comment:
(same comment as the SSM examples) Add more description to explain why you use these examples and how it works.